build.sh

```
@@ -23,11 +25,26 @@
-	echo "Quiet mode enabled."
+	# echo "Quiet mode enabled."
 	BUNFLAGS="$BUNFLAGS --silent --quiet"
 	CARGOFLAGS="$CARGOFLAGS --quiet"
 	export BUN_DEBUG_QUIET_LOGS=1
 fi
+# ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+if [[ "$*" == *"--help"* ]]; then
+	echo "Usage: ./build.sh [options]"
+	echo "Options:"
+	echo "  --help\n\t\tDisplay this help message."
```

Shellcheck

[info] echo may not expand escape sequences. Use printf.

(SC2028)